### DISCUSSION TITLE
Добавление модели

### DISCUSSION THREAD

**Discussion by HepoH3** [Arkady23/V380-Web#5] [Ideas]
Здравствуйте.
Не могли бы вы добавить поддержку модели `HwV380E22_WF3_MINI_Q7`?

**Comment by Arkady23** [comment]
Нужно еще знать hwname / soft-id. Мне кажется должно быть что-то такое V380E3_C2#YW. Я сделал для пробы. Если установится, значит угадал.
https://github.com/Arkady23/V380-Web/releases/download/20210530/httpd_V380E3_C2.YW_v20210614.zip (удалил).
Если не угадал, то вам надо как-то узнать.... самый простой способ — это прослушать трафик от камеры при нажатии на телефоне в приложении пункта "версия прошивки". Если подойдет установщик — сообщите. Так же было бы полезно дать какую-нибудь ссылку на эту камеру на aliexpress или куда-нибудь, где она есть.

**Reply by HepoH3** [reply]
Попробовал, кажется не заработало, по адресу ничего не открывается, cgi-bin в корне не появился (вы про это на 4pda писали).
По поводу hwname/softid, это не поможет?:
```
[kernal_version]
name=KerV380E22_MINI_Q7_V3.1.1
date=20181220
[hardware_version]
name=HwV380E22_WF3_MINI_Q7
date=20201023
[base_version]
name=AppV380E22_V3.3.2.9#YW01
```

**Reply by Arkady23** [reply]
Если у вас есть доступ к version.ini, так может быть есть доступ к patchmanage.conf ??? (Он в той же папке, что и version.ini). Там находится этот код.

**Reply by Arkady23** [reply]
Пробуйте https://github.com/Arkady23/V380-Web/releases/download/20210530/httpd_V380E3_C5.YW_v20210617.zip
Linux-независимая версия httpd с библиотеками.

**Reply by HepoH3** [reply]
> 
> 
> Если у вас есть доступ к version.ini, так может быть есть доступ к patchmanage.conf ??? (Он в той же папке, что и version.ini). Там находится этот код.

Не, это данные из файла quick_check_dump, который записывается при размещении quick_check.ini. Доступа к описанным выше файлам нет.
[quick_check_dump_43879669.txt](https://github.com/Arkady23/V380-Web/files/6676058/quick_check_dump_43879669.txt)
Версия из комментария выше, к сожалению, так же не сработала.

**Reply by Arkady23** [reply]
Как вы определяете, что не сработала. На SD не появляется cgi-bin? Сейчас в версии выше еще должна появиться папка bin

**Reply by Arkady23** [reply]
Если эти папки появляются, то всё должно работать. Если нет, то можно сделать патч, конкретно под вашу версию. Тогда нужна точный номер версии прошивки. Это 7 цифр, например 2040318.

**Reply by HepoH3** [reply]
> 
> 
> Как вы определяете, что не сработала. На SD не появляется cgi-bin? Сейчас в версии выше еще должна появиться папка bin

На sd-карте не появляется ничего нового, попытки открыть в браузере адрес камеры ни к чему ни приводят.
Хм, в версии прошивки у меня 8 цифр (20201023), и судя по всему, это дата её релиза.
![photo_2021-06-18_14-27-48](https://user-images.githubusercontent.com/17159587/122554213-5e641100-d041-11eb-89a6-3134dce9a6c9.jpg)

**Comment by Arkady23** [comment]
Это не версия, это дата. Версия опять-таки в трафике рядом с hwname.

**Reply by HepoH3** [reply]
Не подскажите ли тогда, как снять трафик? Wireshark, полагаю не подойдет, она ведь снимет только трафик между компьютером и роутером, а не между роутером и камерой.

**Reply by Arkady23** [reply]
На главной странице есть ссылка:
Обучающий ролик, как поймать трафик с камеры https://youtu.be/GLYqb2cC4y0.

**Reply by HepoH3** [reply]
Единственное, что нашел похожее в трафике: `AppV380E22_V3.3.2.9#YW01`

![](https://puu.sh/HQ6fO/2737b5dc5e.png)

**Reply by Arkady23** [reply]
Патч с web-интерфейсом на вашу камеру уже есть https://github.com/Arkady23/V380-Web/releases/download/20210530/httpd_V380E3_C5.YW_9090909_v20210620.zip

**Reply by HepoH3** [reply]
> Патч с web-интерфейсом на вашу камеру уже есть https://github.com/Arkady23/V380-Web/releases/download/20210530/httpd_V380E3_C5.YW_9090909_v20210620.zip

Попробовал проверить: скинул содержимое архива в корень отформатированной в FAT32 sd-карты, вставил в камеру, включил питание, подождал минут 10, вбил в браузере ip-адрес — ничего (нет соединения).

**Comment by Arkady23** [comment]
Добавил описание для установки httpd вручную для новых камер.

**Reply by HepoH3** [reply]
Я в итоге настроил запись rstp-потока через свой сервер, так что уже не буду париться с httpd, но все равно спасибо за ваш труд!
